docker/spec/dependabot/docker/update_checker_spec.rb

```diff
@@ -344,7 +344,11 @@ def stub_tag_with_no_digest(tag)
 
       context "and we're using one" do
         let(:version) { "1803" }
-        it { is_expected.to eq("1903") }
+        it "expects another specific version" do
+          VCR.use_cassette("windows-servercore-1803") do
+            expect(subject).to eq("1903")
+          end
+        end
```

Comment:
The second commit feels a bit suspicious, why does this change need extra network requests that need to be recorded? I think it would be the first case of us using VCR here and it would be good to know why, and also probably mock the requests instead of using VCR like the other specs do.

Reply:
I think I figured this out. It was a bug which #8442 should fix. After that PR, the second commit here should not be needed.